Gem/Code/Source/Components/NetworkCharacterComponent.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project
```

Comment:
FYi, these headers got updated and have an additional change, not sure if that will block a PR or not.